pynamodb/tests/test_model.py

```diff
@@ -515,7 +524,7 @@ def test_delete(self):
                 'TableName': 'UserModel'
             }
             args = req.call_args[0][1]
-            self.assertEqual(args, params)
+            deep_eq(args, params, _assert=True)
```

Comment:
a couple of readability suggestions:
- since it's not a library and we can modify the code, why not change ```_assert``` to default to ```True```?
- i would rename it ```assert_deep_eq``` just so it's more obvious to the test reader that it's where the assertion is happening. you could also do ```assert deep_eq(args, params)```. when the assertion passes it will pass and if it fails the inner assertion will still throw the more helpful message